bin/que

```diff
@@ -18,39 +18,39 @@ options = OpenStruct.new
 OptionParser.new do |opts|
   opts.banner = "usage: que [options] file/to/require ..."
 
-  opts.on("-i", "--wake-interval [INTERVAL]", Float, "Set maximum interval between polls of the job queue (in seconds) (default: 0.1)") do |wake_interval|
+  opts.on("-i", "--wake-interval [INTERVAL]", Float, "Set maximum interval between polls of the job queue (in seconds) (default: 5)") do |wake_interval|
     options.wake_interval = wake_interval
   end
 
   opts.on("-p", "--metrics-port [PORT]", Integer, "Enable metrics and expose them on this port") do |metrics_port|
     options.metrics_port = metrics_port
   end
 
-  opts.on("-e", "--cursor-expiry [EXPIRY]", Integer, "Enable performance patch with cached cursors") do |cursor_expiry|
```

Comment:
Not very sure why this was `Integer` initially, since it's a duration like `wake_interval` and `timeout`.